core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -199,11 +227,11 @@ public void serverStarting(Server server) throws Exception {
         });
     }
 
-    private static ServiceSpecification generate(List<ServiceConfig> services, DocServiceFilter filter) {
+    private ServiceSpecification generate(List<ServiceConfig> services) {
         return ServiceSpecification.merge(
                 plugins.stream()
                        .map(plugin -> plugin.generateSpecification(
-                               findSupportedServices(plugin, services), filter))
+                               findSupportedServices(plugin, services), filter, namedTypeInfoProvider))
```

Comment:
Should we assert `namedTypeInfoProvider != null`?
The field is marked `@Nullable` which can be confusing, and if it `null` docservice would break.

Reply:
Removed `@Nullable` annotation from the field declaration.